osi_sensorviewconfiguration.proto

```diff
@@ -913,3 +842,139 @@ message UltrasonicSensorViewConfiguration
+    //
+    // \note The origin of vehicle's coordinate system in world frame is
+    // ( \c MovingObject::base . \c BaseMoving::position +
+    // Inverse_Rotation_yaw_pitch_roll( \c MovingObject::base . \c
+    // BaseMoving::orientation) * \c
+    // MovingObject::VehicleAttributes::bbcenter_to_rear) . The orientation of
+    // the vehicle's coordinate system is equal to the orientation of the
+    // vehicle's bounding box \c MovingObject::base . \c
+    // BaseMoving::orientation. \note A default position can be provided by the
+    // sensor model (e.g. to indicate the position the model was validated for),
+    // but this is optional; the environment simulation must provide a valid
+    // mounting position (based on the vehicle configuration) when setting the
+    // view configuration.
+    //
+    // All ray tracing results e.g. Hitpoint positions are given in this
+    // sensor coordinate system.
+    //
+    optional MountingPosition mounting_position = 2;
+    
+    // Position of the ray tracer`s receiver in the sensor coordinate system.
+    // 
+    repeated MountingPosition receiver_position = 3;
+    
+    // Position of the ray tracer`s transmitter in the sensor coordinate system.
+    //
+    repeated MountingPosition transmitter_position = 4;
+    
+    // The wavelength of the transmitted ray in the simulation.
+    //
+    // Unit: nm
+    optional double emitter_wavelength = 5;
+    
+    // When reaching the maximmum signal loss during ray tracing further ray tracing is aborted.
+    // The maximum signal loss is the relation between current calculated signal strength and the emitted signal
+    // strength at the ray generation.
+    // 
+    // Unit: dB
+    optional double maximum_signal_loss = 6;
+    
+    // The maximum path length desribes the ray's maximum overall distance.
+    // If the maximum path length is reached during simulation no further interaction is calculated.
+    //
+    // Unit: m
+    optional double max_path_length = 7;
+    
+    // The maximum number of interaction desribes the ray's maximum hits with surfaces.
+    // If the maximum number of interaction is reached during simulation no further interaction is calculated.
+    //
+    // Unit: -
+    optional uint32 max_number_of_interactions = 8;
+    
+    // Format of ray tracer data (includes number, kind and format of channels).
+    //
+    //
+    // In the message provided to the sensor model, this field must
+    // contain exactly one value, indicating the format of the image
+    // data being provided by the simulation environment - which must
+    // be one of the values the sensor model requested - or there
+    // must be no value, indicating that the simulation environment
+    // cannot provide ray tracer data in one of the requested formats.
+    //
+    optional RayTracerFormat ray_tracer_format = 9;
+
+    // Different predefined ray tracer formats
+    // 
+    enum RayTracerFormat
```

Comment:
I do not agree to have a raytracer format for every project. What's next? GaiaX format, Pegasus format? We need to come to an agreement here